keel-core/src/test/kotlin/com/netflix/spinnaker/keel/enforcers/EnvironmentExclusionEnforcerTest.kt

```diff
@@ -31,15 +74,14 @@ internal class EnvironmentExclusionEnforcerTest {
     }
   }
 
-  @kotlinx.coroutines.ExperimentalCoroutinesApi
   @ParameterizedTest(name="withActuationLease executes action, enabled={0}")
   @ValueSource(booleans = [true, false])
   fun `withActuationLease invokes the action whether the feature flag is enabled or not`(enabled: Boolean) {
     every { springEnv.getProperty("keel.enforcement.environment-exclusion.enabled", Boolean::class.java, any())} returns enabled
 
     val action : suspend () -> Unit = mockk(relaxed=true)
-    runBlockingTest {
```

Comment:
Was getting an occasional error with `runBlockingTest`, so switched to `runBlocking` instead.

Reply:
I've never seen `runBlockingTest` before. I'll have to look into that

Reply:
It's like runBlocking, but it does things like ignore delays.